File: Java/fib.java
public class fib {
    public static int fib(int n) {
        if (n < 2) return n;
        return fib(n-1) + fib(n-2);
    }
    public static void main(String[] args) {
        System.out.print(fib(5));
    }
}

File: C#/fib.cs
using System;

namespace fib {
    class fib {
        static void Main(string[] arg) {
            Console.Write(fib(5));
        }
        static int fib(int n) {
            if (n < 2) return n;
            return fib(n-1) + fib(n-2);
        }
    }
}